tests/unittests/test_logger_for_events.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from unittest import mock
+from datetime import datetime
+from tap_stripe import Context, sync_stream
+
+class MockClass():
+    '''The mock class for the Balance Transactions/events object.'''
+    lines = "lines"
+    def __init__(self):
+        return None
+
+    @classmethod
+    def to_dict_recursive(cls):
+        '''The mocked to_dict_recursive method of the Balance Transactions/Events class.'''
+        return "Test Data"
+
+BOOKMARK_TIME = 1645046000 # epoch bookmark time
+BOOKMARK_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+@mock.patch("tap_stripe.LOGGER.warning")
+@mock.patch("singer.write_record")
+@mock.patch('singer.utils.now', return_value = datetime.strptime("2022-05-01T08:30:50Z", BOOKMARK_FORMAT))
+@mock.patch("tap_stripe.reduce_foreign_keys", return_value = {"created": 16452804585})
+@mock.patch("tap_stripe.convert_dict_to_stripe_object", return_value = {"created": "2022-02-17T00:00:00"})
+@mock.patch("tap_stripe.paginate", return_value = [MockClass()])
+@mock.patch("tap_stripe.Context.get_catalog_entry")
+@mock.patch("tap_stripe.singer.metadata.to_map")
+@mock.patch("tap_stripe.singer.metadata.get", return_value = ["created"])
+@mock.patch("tap_stripe.epoch_to_dt")
+@mock.patch("tap_stripe.dt_to_epoch", side_effect = [1645056000, 1645056000, 1647647700, 1645056000]) # epoch timestamps
+@mock.patch("tap_stripe.sync_sub_stream")
+@mock.patch("tap_stripe.singer.get_bookmark", side_effect = [BOOKMARK_TIME, BOOKMARK_TIME])
```

Comment:
Can't we keep `return_value =  BOOKMARK_TIME` instead of the side_effect list if it same everytime?